lib/runners/processor/golangci_lint.rb

```diff
@@ -96,10 +96,9 @@ def analyzer_options
         opts << "--color=never"
         opts << "--concurrency=2"
 
-        # NOTE: The value should be less than the top-level timeout value.
-        #
-        # @see https://github.com/sider/runners/blob/1231de0ca047c7a23449ab1a7bb0751f39f16643/images/Dockerfile.end.erb#L11
-        opts << "--timeout=15m"
+        # NOTE: Set `--timeout` to prevent a user-defined timeout value via `.golangci.yml`.
+        #       This value should be less than the top-level timeout value `ENV['RUNNERS_TIMEOUT']`.
```

Comment:
This comment looks unnecessary (and incorrect).
```suggestion
```

Reply:
I think it needed to explain why this value is `10h`. What idea do you have?

Reply:
Oh, `less` is incorrect as you said. So how about this?

```suggestion
        #       This value should be greater than the top-level timeout value `ENV['RUNNERS_TIMEOUT']`.
```

Reply:
I want to suggest adding "enough."

```suggestion
        #       This value should be enough greater than the top-level timeout value `ENV['RUNNERS_TIMEOUT']`.
```

Reply:
nice 👍